src/components/Button.js

```diff
@@ -1,16 +1,19 @@
+/* eslint-disable react/prop-types */
```

Comment:
|REQUIRED] With the ESLint inline comment, you are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead. Please don't disable it.